Sources/SmileID/Classes/Networking/Models/v2/JobType.swift .swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+public enum JobTypeV2: String, Codable {
```

Comment:
Is it possible to name this just JobType? Or would this cause conflicts

Reply:
Causes conflicts